src/ptz-onvif.cpp

```diff
@@ -466,9 +466,9 @@ OnvifDeviceService::OnvifDeviceService()
 		"xmlns:tt=\"http://www.onvif.org/ver10/schema\"");
 }
 
-DeviceCapabilities OnvifDeviceService::GetCapabilities(QString deviceXAddress,
-						       QString username,
-						       QString password)
+OnvifDeviceCapabilities
+OnvifDeviceService::GetCapabilities(QString deviceXAddress, QString username,
+				    QString password)
 {
```

Comment:
Since the ONVIF rename is a self contained change, I've got ahead and applied it as a separate commit.